headapps/nextjs-starter/src/components/Promo.tsx

```diff
@@ -55,24 +108,53 @@ export const Default = (props: PromoProps): JSX.Element => {
   return <PromoDefaultComponent {...props} />;
 };
 
-export const WithText = (props: PromoProps): JSX.Element => {
+export const WithPlaceHolder = (props: PromoProps): JSX.Element => {
   const id = props.params.RenderingIdentifier;
+  const ShowCurvedTop = props.params.ShowCurvedTop === '1';
+  const ShowBlobAccent = props.params.HideBlobAccent !== '1';
   if (props.fields) {
     return (
-      <div className={`component promo ${props?.params?.styles}`} id={id ? id : undefined}>
-        <div className="component-content">
-          <div className="field-promoicon">
-            <JssImage field={props.fields.PromoIcon} />
-          </div>
-          <div className="promo-text">
-            <div>
-              <div className="field-promotext">
-                <JssRichText className="promo-text" field={props.fields.PromoText} />
-              </div>
+      <div
+        className={`component promo bg-background-secondary dark:bg-background-secondary-dark ${props?.params?.styles} relative overflow-hidden group`}
+        id={id ? id : undefined}
+      >
+        {/* Curved Top SVG */}
+        {ShowCurvedTop && (
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 1600 197.47"
+            className="absolute top-0 left-0 w-full h-auto z-1"
+            aria-hidden="true"
+            focusable="false"
+          >
+            <path
+              d="M0,0V2S1589.99-14.42,1599.94,197.47h.06V0H0Z"
+              className="dark:fill-background-dark fill-background"
+            />
+          </svg>
+        )}
+
+        <div className="component-content container grid lg:grid-cols-2 gap-10 pt-15 md:pt-30 pb-10 relative z-2">
+          {/* Blob Accent SVG*/}
+          {ShowBlobAccent && (
+            <div className="absolute -left-30 top-0 z-0 pointer-events-none w-96 h-96 max-w-3xl max-h-3xl">
+              <StrippedBlobAccent />
             </div>
-            <div className="field-promotext">
-              <JssRichText className="promo-text" field={props.fields.PromoText2} />
+          )}
+          <div className="relative order-1 group-[.image-right]:order-1 lg:group-[.image-right]:order-2">
+            <JssImage
+              field={props.fields.PromoIcon}
+              className="w-full h-full rounded-xl object-cover mb-2 lg:mb-0 relative z-10"
+            />
+          </div>
+          <div className="self-center order-2 group-[.image-right]:order-2 lg:group-[.image-right]:order-1">
+            <div className="text-4xl font-bold mb-10">
```

Comment:
wrap this in a heading tag for semantics (h2 preferably). Same for the other variant